.github/workflows/ci.yml

```diff
@@ -14,10 +14,10 @@ jobs:
       AUTH0_API_AUDIENCE: ${{ secrets.AUTH0_API_AUDIENCE }}
     strategy:
       matrix:
-        node: [14, 16, 18, 20]
+        node: [20]
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
```

Comment:
interesting that these weren't bumped by dependabot

Reply:
Agreed. Not sure why that is.